CTestRun.cmake

```
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.16)
+
+set(CTEST_BINARY_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/build)
+set(CTEST_SOURCE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+
+if(NOT "${GITHUB_PR_ID}" STREQUAL "")
```

isn't it equivalent to
```suggestion
if(GITHUB_PR_ID)
```

---

Probably, but I alwyays have to convince myself what happens if the variable is defined but empty which is precisely what I'm doing now for nightly builds. I don't have that doubt when comparing strings.